test/test_validation.py

```diff
@@ -206,7 +207,9 @@ def test_load_xml(self):
         - Properties with undefined dtypes
         """
 
-        doc = odml.load("./resources/integration.xml")
+        dir_path = os.path.dirname(os.path.realpath(__file__))
```

Comment:
Just a minor comment: `dir_path = ...` could be moved to the `setUp()` method and used from there.